src/utils.ts

```diff
@@ -64,13 +66,43 @@ export const buildTransaction = (
     if (flags.multiPassphrases) {
         const passphrases: string[] = flags.multiPassphrases.split(";");
 
+        builder.senderPublicKey(
+            Identities.PublicKey.fromMultiSignatureAsset({
+                min: flags.min || 2,
+                publicKeys: passphrases.map((passphrase: string) => Identities.PublicKey.fromPassphrase(passphrase)),
+            }),
+        );
+
         for (let i = 0; i < passphrases.length; i++) {
             builder.multiSign(passphrases[i], i);
         }
+
+        builder.sign(passphrases[0]);
+
+        // todo
```

Comment:
Only type 4 needs to be signed, normal multisignature transactions only have `signatures` which means the `verify()` below will fail since that doesn't take them into account.